titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java

```diff
@@ -153,10 +172,21 @@ private Subscription observeJob() {
 
         return context.getJobManagementClient()
                 .killTask(taskId, false)
-                .onErrorResumeNext(e -> Observable.error(new IOException("Failed to kill task " + taskId + " of job " + name, e)))
+                .onErrorResumeNext(e -> Observable.error(new IOException(String.format("Failed to kill task %s  of job %s: error=%s", taskId, name, e.getMessage()), e)))
```

Comment:
extra space after `%s`. also, shouldn't the jobId also be in here. There could be hundreds of jobs with the same name.

Reply:
Fixed

Reply:
The name field is unique (it is not application name).